src/test/java/it/mulders/mcs/search/printer/CoordinatePrinterTest.java

```diff
@@ -34,7 +36,7 @@ class CoordinatePrinterTest implements WithAssertions {
             </dependency>
             """;
     private static final String GRADLE_GROOVY_OUTPUT = "implementation group: 'org.codehaus.plexus', name: 'plexus-utils', version: '3.4.1'";
-    private static final String GRADLE_GROOVY_SHORT_OUTPUT = "'org.codehaus.plexus:plexus-utils:3.4.1'";
+    private static final String GRADLE_GROOVY_SHORT_OUTPUT = "implementation 'org.codehaus.plexus:plexus-utils:3.4.1'";
```

Comment:
Good catch - I'm not surprised you're the one who found it :-). But could you please keep it out of this PR?

I don't mind fixing it myself, but now it sneaks along with a new feature.

Reply:
Totally! I'll submit a new PR for this.

Reply:
Thanks, appreciated!